test/python/algorithms/test_measure_error_mitigation.py

```diff
@@ -332,7 +332,7 @@ def test_measurement_error_mitigation_with_vqe_ignis(self, config):
 
     @unittest.skipUnless(HAS_AER, "qiskit-aer is required for this test")
     @unittest.skipUnless(HAS_IGNIS, "qiskit-ignis is required to run this test")
-    def test_callibration_results(self):
+    def test_calibration(self):
```

Comment:
It would be good to throw in a very specific regression test of the behaviour that this fixes - this test doesn't directly test that the `qc_meas` list is unmodified after the run, and if it _is_ modified, because this test doesn't cleanly reset its state between the two versions of `transpiled`, it might cause a spurious failure as well.

Something as simple as:
```python
circuit = QuantumCircuit(1)
circuit.x(0)
circuits = [circuit]
QuantumInstance(...).execute(circuits)
self.assertEqual(circuits, [circuit])
```
with a note that it's a regression test of 7449.